src/application/App.tsx

```diff
@@ -150,26 +135,22 @@ export const App = () => {
 
   useEffect(() => {
     if (clients.length) {
-      send({ type: "connect" });
+      send({ type: "client.setCount", count: clients.length });
     }
   }, [send, clients.length]);
 
   return (
     <>
       <SettingsModal open={settingsOpen} onOpen={setSettingsOpen} />
-      <ClientNotFoundModal
-        open={snapshot.context.modalOpen}
-        onClose={() => send({ type: "closeModal" })}
-        onRetry={() => send({ type: "retry" })}
-      />
+      <Modals />
```

Comment:
Can we name this something like `ErrorModals`? Ideally we have some namespace that describes that these are the modals opened when the clients can't connect in some way. 

Otherwise it looks a little bit weird to have `SettingsModal` right next to `Modals` 😂.